connector-doc-generator/resources/prompt-template-phase2b.md

```
@@ -0,0 +1,167 @@
+
+```json
+{
+  "field": "value"
+}
+```
+
+</div>
+</details>
+```
+
+{{updateModeNote}}
```

markdownlint-cli2 (0.22.1)

[warning] Heading style
Expected: atx; Actual: setext

(MD003, heading-style)